### PR TITLE
framework/task_manager: Implement task_manager_getinfo_with_pid API

### PR DESCRIPTION
**Pull request by jeongchanKim**
1. framework/task_manager: Implement task_manager_getinfo_with_pid API
  User can get task information by pid which is managed by task manager
2. Add positive and negative TCs for task_manager_getinfo_with_pid()

### CODE REVIEW

File: framework/src/task_manager/task_manager_core.c [hunk 1]
```diff
@@ -1202,6 +1202,10 @@ int task_manager(int argc, char *argv[])
 		case TASKMGRCMD_DEALLOC_BROADCAST_MSG:
 			ret = taskmgr_dealloc_broadcast_msg(*((int *)request_msg.data));
 			break;
+
+		case TASKMGRCMD_GET_HANDLE:
```
Comment (sunghan-chang) [inline framework/src/task_manager/task_manager_core.c]: What is difference between this and TASKMGRCMD_SCAN_*?

File: framework/include/task_manager/task_manager.h [hunk 2]
```diff
@@ -423,6 +423,18 @@ int task_manager_unset_broadcast_cb(int msg, int timeout);
...
+ * @details @b #include <task_manager/task_manager.h>
+ * @param[in] pid the pid which to get its handle\n
+ * @param[in] timeout returnable flag. It can be one of the below.\n
+ *			TM_RESPONSE_WAIT_INF : Blocked until get the response from task manager\n
+ *			integer value : Specifies an upper limit on the time for which will block in milliseconds\n
+ *            For this API, timeout cannot be set to TM_NO_RESPONSE.
+ * @return On success, OK is returned. On failure, defined negative value is returned.
+ * @since TizenRT v2.0 PRE
+ */
+int task_manager_get_handle(int pid, int timeout);
```
Comment (sunghan-chang) [inline framework/include/task_manager/task_manager.h]: Does this API need timeout?

File: framework/src/task_manager/task_manager_get_handle.c [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+#include <string.h>
+#include <stdio.h>
+#include <sys/types.h>
+#include <task_manager/task_manager.h>
+#include "task_manager_internal.h"
+
+/****************************************************************************
+ * task_manager_get_handle
+ ****************************************************************************/
+int task_manager_get_handle(int pid, int timeout)
```
Comment (sunghan-chang) [inline framework/src/task_manager/task_manager_get_handle.c]: Is this totally different from getinfo?

**Comment by seinfra** [comment]
Target : [c60da495d4d989fbde079cb64f5591dcf5930bdc] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [c60da495d4d989fbde079cb64f5591dcf5930bdc] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [6da142153be1f4349bd01c8c458aee944d14b601] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6da142153be1f4349bd01c8c458aee944d14b601] - Code Rule Check OK.

File: framework/include/task_manager/task_manager.h [hunk 4]
```diff
@@ -423,6 +423,18 @@ int task_manager_unset_broadcast_cb(int msg, int timeout);
  * @since TizenRT v2.0 PRE
  */
 int task_manager_dealloc_broadcast_msg(int msg, int timeout);
+/**
```
Comment (sunghan-chang) [inline framework/include/task_manager/task_manager.h]: Let's move this near another getinfo function.

File: framework/include/task_manager/task_manager.h [hunk 5]
```diff
@@ -423,6 +423,18 @@ int task_manager_unset_broadcast_cb(int msg, int timeout);
...
+ * @brief Get the handle through pid
+ * @details @b #include <task_manager/task_manager.h>
+ * @param[in] pid the pid which to get its handle\n
```
Comment (sunghan-chang) [inline framework/include/task_manager/task_manager.h]: its task manager information

File: framework/src/task_manager/task_manager_core.c [hunk 6]
```diff
@@ -1202,6 +1202,14 @@ int task_manager(int argc, char *argv[])
 		case TASKMGRCMD_DEALLOC_BROADCAST_MSG:
 			ret = taskmgr_dealloc_broadcast_msg(*((int *)request_msg.data));
 			break;
+
+		case TASKMGRCMD_SCAN_PID:
```
Comment (sunghan-chang) [inline framework/src/task_manager/task_manager_core.c]: same as previos. move near another scan

**Comment by seinfra** [comment]
Target : [905023b41cf10a33636003f3ccce2814ea182f6b] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [905023b41cf10a33636003f3ccce2814ea182f6b] - Code Rule Check OK.

**Pull request merged**